src/CheckoutSdk/Payments/Links/PaymentLinkDetailsResponse.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Checkout.Common;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+namespace Checkout.Payments.Links
+{
+
+    public sealed class PaymentLinkDetailsResponse : Resource, IEquatable<PaymentLinkDetailsResponse>
+	{
+
+		public string Id { get; set; }
```

Comment:
The code is formatted with TABs. The indentation shows up really strange. Can you please reformat the code to avoid them?